src/test/java/create_flashcard_use_case/FcCInteractorTest.java

```diff
@@ -41,13 +44,15 @@ public FcCResponseModel prepareSuccessView(FcCResponseModel responseModel){
                 return null;
             }
 
+            //Errors should not happen.
```

Comment:
Good to include this comment so that the reader knows what should and shouldn't happen in the test.